libtmux/common.py

```diff
@@ -668,3 +668,16 @@ def handle_option_error(error):
         raise exc.AmbiguousOption(error)
     else:
         raise exc.OptionError(error)  # Raise generic option error
+
+
+def get_libtmux_version():
+    """Return libtmux version is a PEP386 compliant format.
+
+    Returns
+    -------
+    distutils.version.LooseVersion
```

Comment:
https://github.com/pypa/distutils/issues/19 This is where i'd like to link to `LooseVersion`